--- a/nixos/tests/bcachefs.nix
+++ b/nixos/tests/bcachefs.nix
@@ -1,8 +1,8 @@
-{ pkgs, ... }:
+{ config, ... }:
 {
   name = "bcachefs";
   meta = {
-    inherit (pkgs.bcachefs-tools.meta) maintainers;
+    inherit (config.node.pkgs.bcachefs-tools.meta) maintainers;
   };
 
   nodes.machine =
--- a/nixos/tests/ringboard.nix
+++ b/nixos/tests/ringboard.nix
@@ -6,7 +6,7 @@
 }:
 {
   name = "ringboard";
-  meta.maintainers = pkgs.ringboard.meta.maintainers ++ (with lib.maintainers; [ h7x4 ]);
+  meta.maintainers = config.node.pkgs.ringboard.meta.maintainers ++ (with lib.maintainers; [ h7x4 ]);
 
   nodes.machine = {
     imports = [
